test/common/common/BUILD

```diff
@@ -10,13 +10,6 @@ load(
 
 envoy_package()
 
-envoy_cc_fuzz_test(
```

Comment:
Hello @htuch, fixed merge conflicts and made sure there are no extra braces. 
Adding this test gives me this error "  //test/common/common:base64_fuzz_test_lib: invalid label 'base64_corpus/.!31337!favicon.ico' in element 0 of attribute 'data' in 'cc_library' rule: invalid target name 'base64_corpus/.!31337!favicon.ico': target names may not contain '!'
 ". Any idea how to fix this? Thanks in advance!

Reply:
Look at your diff below, it looks like you're trying to check-in some weird stuff like `test/common/common/base64_corpus/.!31337!favicon.ico`. Can you `git rm` these from your repo and try again? We do need to preserve this target :)